### ISSUE TITLE
Time interval and 'begin-sentence' context

### ISSUE THREAD

**Issue by gmarty** [phensley/cldr-engine#9]
This is more a question than a bug report.
In a time interval, when I use the default 'middle-of-text' context with the `fr` language, this is what I get:
```js
cldr.Calendars.formatDateInterval(
  start,
  end,
  {
    skeleton: 'yMMMEd',
    context: 'middle-of-text',
  }
)
```
```
mar. 1 janv. – mar. 31 déc. 1963
```

Now, with context set to 'begin-sentence', I get the following:
```
Mar. 1 janv. – Mar. 31 déc. 1963
```

I would expect 'begin-sentence' to return something like this:
```
Mar. 1 janv. – mar. 31 déc. 1963
```
With only the first part of the interval capitalised, and the second part in lower case.

Is this the intended behaviour of CLDR?

**Comment by phensley** [comment]
@gmarty Thanks for the report, I've confirmed this bug and am working on a fix.

**Comment by phensley** [comment]
This should be resolved in version 1.7.3

**Comment by gmarty** [comment]
Wonderful! Thank you
